sdk/src/contexts/solana/utils/tokenBridgeRelayer/instructions/transferWrappedTokensWithRelay.ts

```diff
@@ -14,10 +14,10 @@ import {
   deriveRegisteredTokenAddress,
 } from '../accounts';
 import { getAssociatedTokenAddressSync } from '@solana/spl-token';
-import { getWrappedMeta } from '../../tokenBridge';
 import { BN } from '@project-serum/anchor';
 import { ChainId } from 'types';
 import { deriveSignerSequenceAddress } from '../accounts/signerSequence';
+import { getWrappedMeta } from '@certusone/wormhole-sdk/lib/esm/solana/tokenBridge';
```

Comment:
This fixes an issue where the `getWrappedMeta` from the connect SDK was being used instead of the one from the wormhole-sdk. The former wasn't updated to handle the new Solana contracts. Automatic transfers from Solana in testnet were failing - this change fixes that.